scripts/pylib/twister/twisterlib/reports.py

```diff
@@ -441,31 +446,29 @@ def synopsis(self):
 
     def summary(self, results, unrecognized_sections, duration):
         failed = 0
-        run = 0
+        built_only = 0
         for instance in self.instances.values():
-            if instance.status == "failed":
+            if instance.status == Status.FAIL:
                 failed += 1
             elif instance.metrics.get("unrecognized") and not unrecognized_sections:
                 logger.error("%sFAILED%s: %s has unrecognized binary sections: %s" %
                              (Fore.RED, Fore.RESET, instance.name,
                               str(instance.metrics.get("unrecognized", []))))
                 failed += 1
+            elif instance.status == Status.NOTRUN:
+                built_only += 1
```

Comment:
is there another situation also possible when `notrun` is because of orphaned/failed execution ?

Reply:
no.